configure.ac

```diff
@@ -2206,6 +2228,11 @@ AC_ARG_ENABLE([ed448],
     [ ENABLED_ED448=$enableval ],
     [ ENABLED_ED448=no ]
     )
+AC_ARG_ENABLE([ed448-stream],
```

Comment:
Did you consider using `--enable-ed448=stream` and document in ed448 help?

Reply:
I used an optioning scheme consistent with Sean's for AES streaming -- seems OK?  Maybe better the `=stream` way?  I've tended toward the latter with other new features, FWIW.

Reply:
left as-is, as discussed, to be revisited later (also re `--enable-aesgcm-stream`).